realm/realm-library/src/objectServer/java/io/realm/permissions/PermissionChange.java

```diff
@@ -18,6 +18,7 @@
 import java.util.Date;
 import java.util.UUID;
 
+import edu.umd.cs.findbugs.annotations.SuppressFBWarnings;
```

Comment:
Really? That might be a big problem. Some consumers of Realm run with zero-warning policies.